siliconcompiler/core.py

```diff
@@ -1306,7 +1311,9 @@ def _find_files(self,
         elif len(keypath) >= 5 and keypath[0] == 'tool' and keypath[4] == 'script':
             tool = keypath[1]
             task = keypath[3]
-            refdirs = self._find_files('tool', tool, 'task', task, 'refdir', step=step, index=index)
+            refdirs = self._find_files('tool', tool, 'task', task, 'refdir',
+                                       step=step, index=index,
+                                       abs_path_only=True)
```

Comment:
Why do you need `abs_path_only` here?

Reply:
This is building the search path which still needs to be abspath. We only want the final result to be relative.